os_migrate/tests/unit/test_network.py

```diff
@@ -162,3 +163,24 @@ def test_network_sdk_params(self):
         # disallowed params when creating a network
         self.assertNotIn('availability_zones', sdk_params)
         self.assertNotIn('revision_number', sdk_params)
+        print(sdk_params)
+        print('asdlfasdf')
+        print(refs)
+        print('asdfasdfasdfasdfasdfaf')
+        print(net)
+
+    def test_network_serialization_when_dns_domain_is_false(self):
+        sdk_net = sdk_network()
+        net = Network.from_sdk(None, sdk_net)  # conn=None
+        params, info = net.params_and_info()
+        
+        with mock.patch.dict(params['dns_domain'], {'dns_domain': ''}):
+            self.assertEqual(params['dns_domain'], 'example.org')
+
+    def test_network_import_when_dns_domain_is_false(self):
+        net = Network.from_data(serialized_network())
```

Comment:
I think we need to save the serialized_network() result to a variable, which will allow us to set the dns_domain to empty string, and only then we feed it to Network.from_data, and we can then validate that it is not present in sdk_params.

Reply:
Ahhhh looks like I was over complicating! - Thanks